packages/kubernetes-1.15/kubernetes-tmpfiles.conf

```
@@ -0,0 +1 @@
+d /etc/kubernetes/static-pods - - - -
```

`/etc/kubernetes/manifests` is the path used by kubeadm, so it might make sense to have that for consistency with the rest of the ecosystem.

Rather than change anything elsewhere I'd just add a symlink here:
```suggestion
d /etc/kubernetes/static-pods - - - -
L /etc/kubernetes/manifests - - - - static-pods
```